lib/Frontend/ArgsToFrontendInputsConverter.cpp

```diff
@@ -36,6 +36,13 @@ ArgsToFrontendInputsConverter::ArgsToFrontendInputsConverter(
       FilelistPathArg(args.getLastArg(options::OPT_filelist)),
       PrimaryFilelistPathArg(args.getLastArg(options::OPT_primary_filelist)) {}
 
+void ArgsToFrontendInputsConverter::takeConfigurationFileBuffers(
+    SmallVectorImpl<std::unique_ptr<llvm::MemoryBuffer>> &buffers) {
+  std::move(ConfigurationFileBuffers.begin(), ConfigurationFileBuffers.end(),
+            std::back_inserter(buffers));
+  ConfigurationFileBuffers.clear();
+}
```

Comment:
Here's something I don't know about these data structures: if you move stuff out of ConfigurationFileBuffers, why do you also need to clear them afterwards?

Reply:
`std::move` works on an arbitrary range, so it can't actually resize the container itself. It leaves behind null `std::unique_ptr`s instead. The `clear` isn't strictly necessary since we throw away the converter immediately after this, but if we *were* to use it for something else we wouldn't want to have a bunch of nulls lying around.

Reply:
Thanks!